src/static/css/pages/offices.less

```diff
@@ -32,6 +32,17 @@
             });
         }
     }
+
+    .activity {
```

Comment:
Should these styles be moved out for all activity feeds (especially the date one)?